Sources/CoreCommands/SwiftTool.swift

```diff
@@ -756,13 +756,24 @@ public final class SwiftTool {
             {
                 swiftSDK = targetSwiftSDK
             } else if let swiftSDKSelector = options.build.swiftSDKSelector {
-                swiftSDK = try SwiftSDKBundle.selectBundle(
-                    fromBundlesAt: sharedSwiftSDKsDirectory,
-                    fileSystem: fileSystem,
-                    matching: swiftSDKSelector,
-                    hostTriple: hostTriple,
-                    observabilityScope: observabilityScope
-                )
+                do {
+                    swiftSDK = try SwiftSDKBundle.selectBundle(
+                        fromBundlesAt: sharedSwiftSDKsDirectory,
+                        fileSystem: fileSystem,
+                        matching: swiftSDKSelector,
+                        hostTriple: hostTriple,
+                        observabilityScope: observabilityScope
+                    )
+                } catch {
+                    // If a user-installed bundle for the selector doesn't exist, check if the
+                    // selector is recognized as a default SDK.
+                    if let triple = try? Triple(swiftSDKSelector),
+                       let defaultSDK = SwiftSDK.defaultSwiftSDK(for: triple, hostSDK: hostSwiftSDK) {
```

Comment:
IMO it's important to be explicit in variable naming when working with triples
```suggestion
                    if let targetTriple = try? Triple(swiftSDKSelector),
                       let defaultSDK = SwiftSDK.defaultSwiftSDK(for: targetTriple, hostSDK: hostSwiftSDK) {
```

Reply:
agreed, this was taken from L754 but I've changed both instances to use `targetTriple` 👍